src/Expensify.js

```diff
@@ -10,10 +10,9 @@ import SignInPage from './pages/signin/SignInPage';
 import listenToStorageEvents from './libs/listenToStorageEvents';
 import * as ActiveClientManager from './libs/ActiveClientManager';
 import ONYXKEYS from './ONYXKEYS';
-
 import styles from './styles/styles';
 import Log from './libs/Log';
-
+import MigrateOnyx from './libs/MigrateOnyx';
```

Comment:
Normally I would agree, but since there is no `migrateOnyx()` method (it's just `export default function () {
`) then I feel it should match the file name.

Reply:
Or we could alternatively do something like `OnyxMigration.run()`. Mostly, I'd like there to be consistency so it is easy to identify what is a lib/namespace/class and what isn't.

Reply:
Or change the filename to match (sorry, I meant to offer that as a solution too)

Reply:
Sorry, I'm not really following what you want here :D You want the filename changed to `migrateOnyx.js`?

Reply:
Yep, and then to be imported the same. That works :)

Reply:
Cool, thanks!